assets/web/welcome.html

```diff
@@ -2,12 +2,11 @@
   <header>
     <h1 class="mb-0">Welcome to Sunshine!</h1>
     <p class="mb-0 align-self-start">
-      Before Getting Started, write down below these credentials
+      Before Getting Started, we need you to make a new username and password for accessing the Web UI.
     </p>
   </header>
   <div class="alert alert-warning">
-    These Credentials down below are needed to access the rest of the
-    application.<br />
+    These Credentials down below are needed to access Sunshine's Web UI.<br />
```

Comment:
I'd suggest changing this to:

`The credentials below are needed to access Sunshine's Web UI.`